CONTRIBUTING.md

```diff
@@ -1,45 +1,93 @@
 
+We sincerely welcome you to join PyMilvus, build a collaborative, open-source, and open community. If you have already been familiar with the code and usage of PyMilvus, we welcome you to give back to the community, help newcomers, and pass on the open-source, collaborative, and open spirit.
 
-## How can I contribute?
 
-### Development environment
+## Contributions start here!
 
-You are recommended to develop in a virtual environment launched by python environment management tool,
-for example, virtualenv, conda and etc. When your virtual environment is activate, run command `pip install -r requirements.txt`
- to install dependent packages.
+Issues with label [good-first-issue](https://github.com/milvus-io/pymilvus/labels/good%20first%20issue) and [help-wanted](https://github.com/milvus-io/pymilvus/labels/help%20wanted) in PyMilvus are entry-level issues. They are perfect starting points if you are in the process of getting familiar with the project.
 
+If you want some challenge, look at issues with the label [Hacktoberfest](https://github.com/milvus-io/pymilvus/labels/Hacktoberfest).
 
-### Coding Style
-Before submitting a pull request, make sure the coding style is qualified. run command `pylint --rcfile=pylint.conf milvus/client` 
-to check it.
 
+## What contributions can you make?
 
-## Run unit test with code coverage
+If you identify any problems, you can:
+- file an issue to point out the problem
+- provide minimal reproducible steps in the issue (optional)
+- provide solutions to the issue (optional)
+- create a PR to address this issue (optional)
 
-Before submitting your PR, make sure you have run unit test, and your code coverage rate is >= 90%.
+If you are interested in existing problems, you can:
+- in issues with `question` labels, answer questions from others to offer help
+- in issues with `bug`, `improvement`, `enhancement`:
+  - question, reproduce, or provide solutions
+  - create a PR to repair the issue
 
-```shell 
-$ pytest --cov=milvus/client --cov-report=html
-```
+If you want to request more features for PyMilvus, you can:
+- file an issue to describe the new features and explain why
+- provide implementation scheme and test scheme (optional)
+- create a PR to realize the new features (optional)
 
-You may need a milvus server which is running when you run unit test. See more details on [Milvus server](https://github.com/milvus-io/milvus).
+If you are interested in existing PRs, you can:
+- join the code review and offer advice
+- guide newcomers to finish the PR process
 
+Note: the problems, features, and questions mentioned here are not limited to Python code. They also refer to all kinds of documents (technical documents, API references, contributing guide, etc.)
 
-## Update CHANGLOG.md
+## PyMilvus Code Structure
+`docs/`: Contains source codes for documentation except for API Reference and uses sphinx to create documentation. Most documentation are in folder `rst` under `docs/source`.
```

Comment:
`docs/`: Contains source documentation (except for API Reference) that created by sphinx; most documentation are stored as `.rst` files under `docs/source`.